DEVELOPERS.md

```diff
@@ -1,3 +1,7 @@
+# Requirements
+* yarn version 1  (classic)
```

Comment:
nit: would prefer this change is on a separate commit from this PR since they have different intentions